sdk/cosmos/azure-cosmos/azure/cosmos/_change_feed/change_feed_utils.py

```diff
@@ -0,0 +1,155 @@
+
+from azure.cosmos._change_feed.change_feed_state import ChangeFeedMode
+
+def add_args_to_kwargs(
+        args: Tuple[Any, ...],
+        kwargs: Dict[str, Any]
+    ) -> None:
+    """Add positional arguments(args) to keyword argument dictionary(kwargs).
+    Since 'query_items_change_feed' method only allows the following 4 positional arguments in the exact order
+    and types, if the order and types don't match, errors will be raised.
+    If the positional arguments are in the correct orders and types, the arguments will be added to keyword arguments.
+
+    4 positional arguments:
+        - str 'partition_key_range_id': [Deprecated] ChangeFeed requests can be executed against specific partition
+            key ranges. This is used to process the change feed in parallel across multiple consumers.
+        - bool 'is_start_from_beginning': [Deprecated] Get whether change feed should start from
+            beginning (true) or from current (false). By default, it's start from current (false).
+        - str 'continuation': e_tag value to be used as continuation for reading change feed.
+        - int 'max_item_count': Max number of items to be returned in the enumeration operation.
+
+    :param args: Positional arguments. Arguments must be in the following order:
+        1. partition_key_range_id
+        2. is_start_from_beginning
+        3. continuation
+        4. max_item_count
+    :type args: Tuple[Any, ...]
+    :param kwargs: Keyword arguments
+    :type kwargs: dict[str, Any]
+    """
+    if len(args) > 4:
+        raise ValueError(f"Too many arguments. Expected: less than 4, but given: {len(args)}")
+
+    if len(args) > 0:
+        key_and_types = [
+            ('partition_key_range_id', str),
+            ('is_start_from_beginning', bool),
+            ('continuation', str),
+            ('max_item_count', int),
+        ]
+        for i, value in enumerate(args):
+            key, expected_type = key_and_types[i]
+
+            if key in kwargs:
+                raise ValueError(f"'{key}' is in both positional and keyword argument list. Please remove one of them.")
+
+            if not isinstance(value, expected_type):
+                raise TypeError(f"'{value}' is not of type '{expected_type.__name__}'")
+            kwargs[key] = value
+
+def validate_kwargs(
+        kwargs: Dict[str, Any]
+    ) -> None:
+    """Validate keyword arguments(kwargs).
+    The values of keyword arguments must match the expect type and conditions. If the conditions do not match,
+    errors will be raised with the error messages and possible ways to correct the errors.
+
+    :param kwargs: Keyword arguments to verify for query_items_change_feed API
+    :keyword change_feed_mode: Must be one of the values in the Enum, 'ChangeFeedMode'.
+        If the value was 'ALL_VERSIONS_AND_DELETES', the following keywords cannot be used:
+            - 'partition_key_range_id'
+            - 'is_start_from_beginning'
+            - 'start_time'
```

Comment:
if the changeFeedMode is allVersionsAndDeletes, start_time can still be used when the value is "NOW"? and also I the is_start_from_beginning is false?